about.css

```diff
@@ -256,4 +266,54 @@ button:disabled {
   line-height: 52px;
   letter-spacing: 0;
   text-align: left;
+  width: 315px;
+}
+
+@media only screen and (min-width: 768px) {
+  .projects-heading {
+    flex-direction: row;
+    align-items: center;
+    justify-content: space-between;
+  }
+
+  .projects-heading > div {
+    width: 60%;
+    display: flex;
+    align-items: center;
+  }
+
+  .project1-image {
+    padding-right: 24px;
+  }
+
+  .project1-card {
+    flex-direction: row;
+    align-items: start;
+    justify-content: space-between;
+  }
```

Comment:
- [ ] Your design looks very nice :clap: but if you see Figma's design the cards are not the same as your cards. Kindly stick to the Figma design and fix your issue. Also when you hover any card then the overlay will disappear and the button will be shown a comparison of  your design to the Figma design check below :point_down: 

**Your Design**
![Screenshot from 2023-03-30 00-10-24](https://user-images.githubusercontent.com/45226848/228643195-ed40436b-146d-44d4-a041-1247f2de04c7.png)

**Figma design**
![Screenshot from 2023-03-29 23-58-57](https://user-images.githubusercontent.com/45226848/228643239-4a33ddd0-0f37-41b5-a2fa-91f6eed1212c.png)

- [ ] If you see all cards of Figma design you will see an overlay on all the cards except one card and that card you will style if someone is hovering over any of your cards the overlay will disappear and the button will be shown as I mentioned above :point_up: 
- [ ] Check below :point_down:  my video and forget about the design in this video you can just focus on when I hover over any of my cards the button is shown and the overlay is hidden :slightly_smiling_face: 

**[Video](https://user-images.githubusercontent.com/45226848/228644898-c8492779-4293-4033-9286-24438f55c9d7.webm)**

**_Note: You will implement these changes only for the desktop version, not for the mobile version_**